cpp/src/sampling/sampling_post_processing_impl.cuh

```diff
@@ -166,9 +166,14 @@ void check_input_edges(
                                               std::numeric_limits<label_index_t>::max()),
                   "Invalid input arguments: current implementation assumes that the number of "
                   "unique labels is no larger than std::numeric_limits<uint32_t>::max().");
+
+  // FIXME figure out a version of this condition that still allows empty batches
+  /*
   CUGRAPH_EXPECTS(!edgelist_label_offsets || std::get<1>(*edgelist_label_offsets) > 0,
                   "Invlaid input arguments: there should be 1 or more labels if "
                   "edgelist_label_offsets.has_value() is true.");
+  */
```

Comment:
You can just delete this check if you want this function to handle empty batches (or you can just skip calling this function for empty batches). Just make sure that the returned value still makes sense if you call this with an empty batch.

Reply:
There is a Python test specifically for empty batches, so I think we are ok here.